### PR TITLE
New Beneficiary Portal Frontend

### PR DESCRIPTION
**Pull request by OStouraiti**
In progress PR for the beneficiary portal.

Changes so far: 
- confirmation photo and thank you note modals (will need some front-end changes to make sure they show correctly in different screens).
- new item cards
- header and footer
- tabs for the "requested" and "confirm pickup" categories
- completed items section

(contains language select at the bottom, until the hamburger menu is made)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,5 +21,9 @@
     "hooks": {
       "pre-commit": "npx lerna run --stream precommit"
     }
+  },
+  "dependencies": {
+    "react-icons": "^3.10.0",
+    "react-modal": "^3.11.2"
```
Comment (lucashu1) [inline package.json:27]: Remember to move these to the lower-level package.json, not the root one

File: lerna-debug.log [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+8 error MaxBufferError: stdout maxBuffer exceeded
+8 error     at PassThrough.<anonymous> (C:\Users\19176\Desktop\Duet Code\duet\node_modules\get-stream\index.js:41:19)
+8 error     at PassThrough.emit (events.js:322:22)
+8 error     at addChunk (_stream_readable.js:286:12)
+8 error     at readableAddChunk (_stream_readable.js:264:11)
+8 error     at PassThrough.Readable.push (_stream_readable.js:209:10)
+8 error     at PassThrough.Transform.push (_stream_transform.js:152:32)
+8 error     at PassThrough.afterTransform (_stream_transform.js:96:10)
+8 error     at PassThrough._transform (_stream_passthrough.js:46:3)
+8 error     at PassThrough.Transform._read (_stream_transform.js:191:10)
```
Comment (lucashu1) [inline lerna-debug.log:24]: Make sure to remove this from the PR haha

File: packages/backend/src/server/util/auth.js [hunk 3]
```diff
@@ -155,6 +155,7 @@ passport.serializeUser((user, done) => {
   }
   // beneficiary
   if (user.beneficiaryId) {
+    console.log(user);
```
Comment (lucashu1) [inline packages/backend/src/server/util/auth.js:158]: Just a reminder to remove these console.logs before the final PR review! You may have been planning to do this already (I know I'm adding comments before you said it's really ready for review), but I often find it helpful to just scroll through the PR myself to catch stuff like this

**Comment by Raghav-Maheshwari** [comment]
@OStouraiti once this PR is ready to be reviewed, can you add screenshots of all the UI changes in the PR description?

**Pull request merged**